.github/workflows/ci.yml

```diff
@@ -35,14 +42,32 @@ jobs:
       static: ${{ matrix.static != '' && true || false }}
       version: ${{ needs.fetch-version.outputs.version }}
 
+  test-dev-packages-arm64:
+    needs: [fetch-version, build-dev-packages]
+    uses: ./.github/workflows/reusable_test_packages.yaml
+    strategy:
```

Comment:
We can remove the strategy here and remove `static:` below, since we don't yet support `static` builds on arm64 (but i will rebase https://github.com/falcosecurity/falco/pull/2532 and fix it!)

Reply:
After all It is not an issue at all to leave them here anyway!